Problem with TerrainControl plugin.

Hi, I have an issue with TerrainControl plugin.

This is happening when I'm trying to generate a new world with TerrainControl:
``
[20:27:54 INFO]: -------- World Settings For [world] --------
[20:27:54 INFO]: [TerrainControl] [INFO] World world is now enabled!
[20:27:54 ERROR]: Encountered an unexpected exception
net.minecraft.server.v1_8_R3.ReportedException: Exception initializing level
        at net.minecraft.server.v1_8_R3.WorldServer.a(WorldServer.java:852) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:268) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:292) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:559) [NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at java.base/java.lang.Thread.run(Thread.java:829) [?:?]
Caused by: java.lang.NoSuchMethodError: 'net.minecraft.server.v1_8_R3.IBlockData net.minecraft.server.v1_8_R3.Chunk.a(net.minecraft.server.v1_8_R3.BlockPosition, net.minecraft.server.v1_8_R3.IBlockData)'
        at com.khorn.terraincontrol.bukkit.BukkitWorld.setBlock(BukkitWorld.java:446) ~[?:?]
        at com.khorn.terraincontrol.generator.resource.PlantType.spawn(PlantType.java:152) ~[?:?]
        at com.khorn.terraincontrol.generator.resource.GrassGen.spawnNotGrouped(GrassGen.java:137) ~[?:?]
        at com.khorn.terraincontrol.generator.resource.GrassGen.spawnInChunk(GrassGen.java:80) ~[?:?]
        at com.khorn.terraincontrol.generator.resource.Resource.process(Resource.java:67) ~[?:?]
        at com.khorn.terraincontrol.generator.ObjectSpawner.populate(ObjectSpawner.java:70) ~[?:?]
        at com.khorn.terraincontrol.bukkit.generator.TCBlockPopulator.populate(TCBlockPopulator.java:25) ~[?:?]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:341) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.Chunk.loadNearby(Chunk.java:1212) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.originalGetChunkAt(ChunkProviderServer.java:245) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:164) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:124) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.loadChunk(CraftWorld.java:277) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.loadChunk(CraftWorld.java:169) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getHighestBlockYAt(CraftWorld.java:100) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getHighestBlockAt(CraftWorld.java:616) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at com.khorn.terraincontrol.bukkit.generator.TCChunkGenerator.canSpawn(TCChunkGenerator.java:78) ~[?:?]
        at net.minecraft.server.v1_8_R3.WorldServer.canSpawn(WorldServer.java:200) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.WorldServer.b(WorldServer.java:915) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.WorldServer.a(WorldServer.java:837) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        ... 4 more
[20:27:54 ERROR]:       Cause of unexpected exception was
java.lang.NoSuchMethodError: 'net.minecraft.server.v1_8_R3.IBlockData net.minecraft.server.v1_8_R3.Chunk.a(net.minecraft.server.v1_8_R3.BlockPosition, net.minecraft.server.v1_8_R3.IBlockData)'
        at com.khorn.terraincontrol.bukkit.BukkitWorld.setBlock(BukkitWorld.java:446) ~[?:?]
        at com.khorn.terraincontrol.generator.resource.PlantType.spawn(PlantType.java:152) ~[?:?]
        at com.khorn.terraincontrol.generator.resource.GrassGen.spawnNotGrouped(GrassGen.java:137) ~[?:?]
        at com.khorn.terraincontrol.generator.resource.GrassGen.spawnInChunk(GrassGen.java:80) ~[?:?]
        at com.khorn.terraincontrol.generator.resource.Resource.process(Resource.java:67) ~[?:?]
        at com.khorn.terraincontrol.generator.ObjectSpawner.populate(ObjectSpawner.java:70) ~[?:?]
        at com.khorn.terraincontrol.bukkit.generator.TCBlockPopulator.populate(TCBlockPopulator.java:25) ~[?:?]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:341) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.Chunk.loadNearby(Chunk.java:1212) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.originalGetChunkAt(ChunkProviderServer.java:245) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:164) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.ChunkProviderServer.getChunkAt(ChunkProviderServer.java:124) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.loadChunk(CraftWorld.java:277) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.loadChunk(CraftWorld.java:169) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getHighestBlockYAt(CraftWorld.java:100) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at org.bukkit.craftbukkit.v1_8_R3.CraftWorld.getHighestBlockAt(CraftWorld.java:616) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at com.khorn.terraincontrol.bukkit.generator.TCChunkGenerator.canSpawn(TCChunkGenerator.java:78) ~[?:?]
        at net.minecraft.server.v1_8_R3.WorldServer.canSpawn(WorldServer.java:200) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.WorldServer.b(WorldServer.java:915) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.WorldServer.a(WorldServer.java:837) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:268) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:292) ~[NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:559) [NachoSpigot.jar:git-NachoSpigot-"90ec0fb"]
        at java.base/java.lang.Thread.run(Thread.java:829) [?:?]
``

--- a/NachoSpigot-Server/src/main/java/net/minecraft/server/Chunk.java
+++ b/NachoSpigot-Server/src/main/java/net/minecraft/server/Chunk.java
@@ -586,6 +586,10 @@ public class Chunk {
         return this.g(blockposition.getX() & 15, blockposition.getY(), blockposition.getZ() & 15);
     }
 
+    public IBlockData a(BlockPosition blockposition, IBlockData iblockdata) {
+        return a(blockposition,iblockdata,true);
+    }
+
     // KigPaper - add updateLight param
     public IBlockData a(BlockPosition blockposition, IBlockData iblockdata, boolean updateLight) {
         int i = blockposition.getX() & 15;

--- a/NachoSpigot-Server/src/main/java/net/minecraft/server/Chunk.java
+++ b/NachoSpigot-Server/src/main/java/net/minecraft/server/Chunk.java
@@ -586,9 +586,11 @@ public class Chunk {
         return this.g(blockposition.getX() & 15, blockposition.getY(), blockposition.getZ() & 15);
     }
 
+    // NachoSpigot start
     public IBlockData a(BlockPosition blockposition, IBlockData iblockdata) {
         return a(blockposition,iblockdata,true);
     }
+    // NachoSpigot end
 
     // KigPaper - add updateLight param
     public IBlockData a(BlockPosition blockposition, IBlockData iblockdata, boolean updateLight) {

